libraries/chain/include/eosio/chain/thread_utils.hpp

```diff
@@ -341,3 +341,31 @@ namespace eosio { namespace chain {
    }
 
 } } // eosio::chain
+
```

Comment:
Just curious. Why is this put in thread_utils header file?

Reply:
I was asking why it was not in a file in FC directory because I saw it was defined in FC name space.

Reply:
Then no need to change.